src/front-door/azext_front_door/_params.py

```diff
@@ -263,3 +266,59 @@ def load_arguments(self, _):
         c.argument('rule_name', options_list=['--name', '-n'], help='Name of the custom rule.')
         c.argument('policy_name', waf_policy_name_type, id_part=None)
     # endregion
+
+    # region RulesEngine
+    with self.argument_context('network front-door rules-engine rule') as c:
+        c.argument('front_door_name', frontdoor_name_type, options_list=['--front-door-name', '-n'])
+        c.argument('rules_engine_name', rules_engine_name_type, id_part=None)
+        c.argument('rule_name', help='Name of the rule')
```

Comment:
Usually in Azure CLI, the deepest name should support `-n` and `--name`. The `front_door_name` should be just `--front-door-name`

Reply:
hmm.. do you mean the --rules-engine-name should be -n instead?

Reply:
`rule_name` should be `--name/-n`

Reply:
done params changes in next commit